chore(fix) - update config (#107)
dgraph-io/dgraph-docs: content/deploy/config.md

For a single server setup, recommended for new users, please see [Get Started]({{< relref "get-started/index.md" >}}) page.
{{% /notice %}}

> The list of the available subcommands can be viewed by invoking `dgraph --help`.  The full set of configuration options for the subcommand can be viewed by invoking `dgraph <subcommand> --help`, such as `dgraph zero --help`.

Reviewer: ```suggestion
You can see the list of available subcommands with `dgraph --help`.  You can view the full set of configuration options for a given subcommand with `dgraph <subcommand> --help` (for example, `dgraph zero --help`).
```

Reviewer: Edited to remove passive voice

> The options can be configured in multiple ways (from highest precedence to lowest precedence):

Reviewer: ```suggestion
You can configure options in multiple ways, which are listed below from highest precedence to lowest precedence:
```

If no configuration for an option is used, then the default value as described
in the `--help` output applies.

> Multiple configuration methods can be used all at the same time,  e.g. a core

Reviewer: ```suggestion
You can use multiple configuration methods at the same time, so a core
```

set of options could be set in a config file, and instance specific options
could be set using environment vars or flags.

## Command line flags

> Dgraph will have *global flags* that apply to all subcommands and flags specific to a subcommand. Below is an example of using command line flags with `dgraph alpha`.

Reviewer: will have -> has

output. They are the concatenation of `DGRAPH`, the subcommand invoked
(`ALPHA`, `ZERO`, `LIVE`, or `BULK`), and then the name of the flag (in
uppercase). For example, instead of using `dgraph alpha --block_rate 10`, you
could use `DGRAPH_ALPHA_BLOCK_RATE=10 dgraph alpha`.

{{% notice "tip" %}}
For command line flags with the dot `.` notation, you can use underscore `_` in the environment variable name.  Thus `dgraph zero --jaeger.collector` becomes `DGRAPH_ZERO_JAEGER_COLLECTOR`.
{{% /notice %}}

Below is an example of environment variables for `dgraph alpha`:

```bash
DGRAPH_ALPHA_BADGER_COMPRESSION="zstd:1"
DGRAPH_ALPHA_BLOCK_RATE="10"
DGRAPH_ALPHA_CLIENT_AUTH="VERIFYIFGIVEN"
DGRAPH_ALPHA_JAEGER_COLLECTOR="http://jaeger:14268"
DGRAPH_ALPHA_TLS_CACERT="/dgraph/tls/ca.crt"
DGRAPH_ALPHA_TLS_CERT="/dgraph/tls/client.dgraphuser.crt"
DGRAPH_ALPHA_TLS_INTERNAL_PORT_ENABLED="true"
DGRAPH_ALPHA_TLS_KEY="/dgraph/tls/client.dgraphuser.key"
DGRAPH_ALPHA_TLS_NODE_CERT="/dgraph/tls/node.crt"
DGRAPH_ALPHA_TLS_NODE_KEY="/dgraph/tls/node.key"
DGRAPH_ALPHA_TLS_USE_SYSTEM_CA="true"
DGRAPH_ALPHA_WHITELIST="10.0.0.0/8,172.0.0.0/8,192.168.0.0/16"
```

## Configuration file

> A configuration file can be specified using the `--config` flag, or an environment variable, e.g. `dgraph alpha --config my_config.json` or `DGRAPH_ALPHA_CONFIG=my_config.json dgraph alpha`.

Reviewer: e.g. -> such as

> The config file structure is just simple key/value pairs (mirroring the flag names).

Reviewer: ```suggestion
The config consists of a set of key/value pairs, where the keys mirror the flag names).
```

> Configuration file formats supported are [JSON](https://www.json.org/json-en.html), [TOML](https://toml.io/en/), [YAML](https://yaml.org/), [HCL](https://github.com/hashicorp/hcl), and [Java

Reviewer: I'd suggest a construction like the following for lines 83-85:

 "Dgraph supports several configuration file formats that it detects based on file extensions (`.json`, `.toml`,`.yml` or `.yaml`, `.hcl`, and `.properties").
 
 Then, you could link from the file extension referenced here to the 3rd-party site with information about that format.

properties](https://en.wikipedia.org/wiki/.properties) (detected via file extension). The file extensions are `.json`, `.toml`,
`.yml` or `.yaml`, `.hcl`, and `.properties` for each format.

{{% notice "tip" %}}

> For command line flags with the dot `.` notation, you can use the full name as the key, e.g `jaeger.collector`, or represent them hierarchically where the key `jaeger` points to a hash.  For flags in snake case or using underscores, `_`, e.g. `tls_cacert`, you cannot represent them hierarchically.  See language specific examples below for further information.

Reviewer: ```suggestion
For command-line flags with the dot (`.`) notation, you can use the full name as the key (such as `jaeger.collector`), or represent them hierarchically where the key (`jaeger`) points to a hash. For flags in snake case or using underscore (`_`) notation (such as `tls_cacert`), you cannot represent them hierarchically.  See language-specific examples below for further information.
```
